ImageD11/frelon_peaksearch.py

```diff
@@ -360,6 +360,100 @@ def guess_bg(ds, scan_number=0, start=0, step=9, n=None):
 PKCOL = [getattr(ImageD11.cImageD11, p) for p in PKSAVE]
 
 
+def collect_all_frames_peaks(
+    master_file,
+    scans,
+    omega_angles,
+    detector,
+    worker_args,
+    num_cpus=None,
+    **process_map_kwargs,
```

Comment:
Lint for 2.7 seems to dislike the trailing comma, which is only allowed since 3.6: https://test-black.readthedocs.io/en/style-guide/style_guide/trailing_commas.html